Iterable/v1 (#882)
stitchdata/docs: _saas-integrations/iterable-core/v1/iterable-core-v1.md

# -------------------------- #
#      Incompatibilities     #
# -------------------------- #

# -------------------------- #
#      Setup Instructions    #
# -------------------------- #

requirements-list:
  - item: |
      You must be an **org admin** or a user with **API and Webhook configuration permissions** in your {{ integration.display_name }} account.

setup-steps:
  - title: "Generate an {{ integration.display_name }} API key"
    anchor: "generate-api-key"
    content: |
      1. Sign into you {{ integration.display_name }} account.
      2. Navigate to **Integrations**>**API Keys**.
      3. Click **New API Key**.
      4. In the **Create a new API key** window, name your API key and select **read-only**.
      5. Click **Create**.
      6. Copy your new API key and have it ready for the next step.

  - title: "Add {{ integration.display_name }} as a Stitch data source"
    anchor: "add-stitch-data-source"
    content: |
      {% include integrations/shared-setup/connection-setup.html %}
      4. In the **Iterable API Key** field, paste the API key you copied in [step 1](#generate-api-key).

>       5. In the **Window in Days** field, enter a value. This should be set to an optimal value so that historical syncs are completed on time, as setting this value too low may result in historical syncs running for very long time. Setting this value too high could cause the integration to timeout and send `out of memory` errors due to large data flow. This parameter can be reduced to a value as low as 1 day for before syncing latest/recent data.

Reviewer: ```suggestion
      5. In the **Window in Days** field, enter a value. This should be set to an optimal value so that historical syncs are completed on time, as setting this value too low may result in historical syncs running for very long time. Setting this value too high could cause the integration to timeout and send `out of memory` errors due to large data flow. This parameter can be reduced to a value as low as 1 day, after syncing historical data.
```

Reviewer: @jazsmi please make this change.
